meson.build

```diff
@@ -0,0 +1,76 @@
+## Experimetal meson build system for dinit
+project(
+    'dinit', 
+    'cpp', 
+    version : run_command(
+        [find_program('sh'), 'build/general_variables_for_meson.sh', 'version'],
```

Comment:
it's `/bin/sh`. You've already got it hardcoded in the scripts, why search for it here...